helpers/notifications.js

```diff
@@ -1,17 +1,18 @@
 const { ObjectId } = require("mongodb");
 const getMongoConnection = require("./mongoConnect");
 const { Expo } = require("expo-server-sdk");
-const cron = require("node-cron");
+const getToken = require("../middleware/verifyAndAttachJWTData");
```

Comment:
I only saw your PR this morning so I see this isn't necessary now

Reply:
Please remove this import verifyAndAttachJWTData since it is not needed to be used on this file. It already runs right before **attachCurrentUser** runs. (see my other comment)